frontend/dockerfile/instructions/commands.go

```diff
@@ -272,6 +272,13 @@ type RunCommand struct {
 	FlagsUsed []string
 }
 
+func (c *RunCommand) Expand(expander SingleWordExpander) error {
+	if err := SetMountState(c, expander); err != nil {
```

Comment:
don't make this public and call it `expandMounts`.

Reply:
sure thing, but given that this function actually sets the mounts, doesn't `setMountState` make more sense?